frontend/public/components/_catalog-item.scss

```diff
@@ -55,3 +55,7 @@ $catalog-item-icon-size-sm: 24px;
     }
   }
 }
+
+.co-catalog-item-clickable {
+  cursor: pointer;
```

Comment:
This indicates an accessibility problem. It should not be needed.

We should fix the accessibility of these tiles in patternfly

Reply:
I don't want to block the PR on the upstream accessibility problem, but we should remove this style and work with @jeff-phillips-18 on making sure the tiles are accessible

Reply:
fixed